tools/CleanupBuild.ps1

```diff
@@ -2,76 +2,82 @@
-$resourcesFolders = @("de", "es", "fr", "it", "ja", "ko", "ru", "zh-Hans", "zh-Hant", "cs", "pl", "pt-BR", "tr")
-Get-ChildItem -Include $resourcesFolders -Recurse -Force -Path $output | Remove-Item -Force -Recurse -ErrorAction SilentlyContinue
+$outputPaths = @($output)
+if(Test-Path $outputStack)
+{
+    $outputPaths += $outputStack
+}
+foreach ($path in $outputPaths)
+{
+    Write-Verbose "Removing generated NuGet folders from $path"
+    $resourcesFolders = @("de", "es", "fr", "it", "ja", "ko", "ru", "zh-Hans", "zh-Hant", "cs", "pl", "pt-BR", "tr")
+    Get-ChildItem -Include $resourcesFolders -Recurse -Force -Path $path | Remove-Item -Force -Recurse -ErrorAction SilentlyContinue
 
-Write-Verbose "Removing autogenerated XML help files, code analysis, config files, and symbols."
-$exclude = @("*.dll-Help.xml", "Scaffold.xml", "RoleSettings.xml", "WebRole.xml", "WorkerRole.xml")
-$include = @("*.xml", "*.lastcodeanalysissucceeded", "*.dll.config", "*.pdb")
-Get-ChildItem -Include $include -Exclude $exclude -Recurse -Path $output | Remove-Item -Force -Recurse
-Get-ChildItem -Recurse -Path $output -Include *.dll-Help.psd1 | Remove-Item -Force
+    Write-Verbose "Removing autogenerated XML help files, code analysis, config files, and symbols."
+    $exclude = @("*.dll-Help.xml", "Scaffold.xml", "RoleSettings.xml", "WebRole.xml", "WorkerRole.xml")
+    $include = @("*.xml", "*.lastcodeanalysissucceeded", "*.dll.config", "*.pdb")
+    Get-ChildItem -Include $include -Exclude $exclude -Recurse -Path $path | Remove-Item -Force -Recurse
+    Get-ChildItem -Recurse -Path $path -Include *.dll-Help.psd1 | Remove-Item -Force
 
-Write-Verbose "Removing markdown help files and folders"
-Get-ChildItem -Recurse -Path $output -Include *.md | Remove-Item -Force -Confirm:$false
-Get-ChildItem -Directory -Include help -Recurse -Path $output | Remove-Item -Force -Confirm:$false -ErrorAction "Ignore"
+    Write-Verbose "Removing markdown help files and folders"
+    Get-ChildItem -Recurse -Path $path -Include *.md | Remove-Item -Force -Confirm:$false
+    Get-ChildItem -Directory -Include help -Recurse -Path $path | Remove-Item -Force -Confirm:$false -ErrorAction "Ignore"
 
-Write-Verbose "Removing unneeded web deployment dependencies"
-$webdependencies = @("Microsoft.Web.Hosting.dll", "Microsoft.Web.Delegation.dll", "Microsoft.Web.Administration.dll", "Microsoft.Web.Deployment.Tracing.dll")
-Get-ChildItem -Include $webdependencies -Recurse -Path $output | Remove-Item -Force
+    Write-Verbose "Removing unneeded web deployment dependencies"
+    $webdependencies = @("Microsoft.Web.Hosting.dll", "Microsoft.Web.Delegation.dll", "Microsoft.Web.Administration.dll", "Microsoft.Web.Deployment.Tracing.dll")
+    Get-ChildItem -Include $webdependencies -Recurse -Path $path | Remove-Item -Force
+}
 
-$resourceManagerFolders = Get-ChildItem -Path $resourceManagerPath -Directory
-foreach ($RMFolder in $resourceManagerFolders)
+$resourceManagerPaths = @($resourceManagerPath)
```

Comment:
You should move this above with the other if(..) statement so you can do it in one go.

Reply:
Nevermind, it could be possible, but very rare that one folder exists but not the other.